R/helpers.R

```diff
@@ -62,19 +62,19 @@ use_dependency <- function(package, type, min_version = NULL) {
       "!" = "Package {.pkg {package}} is already listed in
              {.field {existing_type}} in DESCRIPTION; no change made."
     ))
-  } else if (delta == 0 && !is.null(min_version)) {
+  } else if (delta == 0 && version_spec(version) != version_spec(existing_version)) {
     # change version
-    upgrade <- existing_version == "*" ||
-      numeric_version(min_version) > version_spec(existing_version)
-    if (upgrade) {
-      ui_bullets(c(
-        "v" = "Increasing {.pkg {package}} version to {.val {version}} in
-               DESCRIPTION."
-      ))
-      desc$set_dep(package, type, version = version)
-      desc$write()
-      changed <- TRUE
-    }
+    direction <- ifelse(version_spec(version) > version_spec(existing_version),
+                        "Increasing", "Decreasing")
```

Comment:
Can you change this to an `if () ... else`? I generally reserve `ifelse()` for vectorized situations.

Reply:
Sure!